modules/nw-metalLB-basic-upgrade-operator.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
```

Comment:
This doesn't render well. There will be two "Upgrading the metallb operator" sections:

![image](https://user-images.githubusercontent.com/200560/178958882-1cd740b3-b7c2-417c-99a5-4aa6756fba08.png)